### DISCUSSION TITLE
Font in Chrome on MacOS Sonoma is not displaying the class Name for each class correctly.

### DISCUSSION THREAD

**Discussion by johnmrichey** [pybricks/support#1544] [General]
In the latest build of Chrome the class names are not being displayed correctly.
<img width="502" alt="Screenshot 2024-03-14 at 00 04 16" src="https://github.com/pybricks/support/assets/63120157/e06369ad-f9dd-4cdf-b6da-56f2b62de8df">

**Comment by laurensvalk** [comment]
Are you using any auto translate feature in your browser, or is this the actual page? Thanks!

**Reply by johnmrichey** [reply]
no translate feature it's the actual page.  On my older Monterey MacOS machine it's displaying fine.

**Reply by johnmrichey** [reply]
<img width="636" alt="Screenshot 2024-03-14 at 00 25 54" src="https://github.com/pybricks/support/assets/63120157/53cf1c95-b303-4812-adb9-df28470ec393">

**Reply by johnmrichey** [reply]
Version 122.0.6261.129 (Official Build) (arm64)

**Reply by laurensvalk** [reply]
Thank you for taking the time to report this. Does it happen for **both** of the following links, or only one?

The version that comes in the app: https://code.pybricks.com/static/docs/v2.16.0/hubs/index.html

The version on "read the docs": https://docs.pybricks.com/en/latest/hubs/index.html

**Reply by johnmrichey** [reply]
Yes, both those links show the class names with just garbage.
<img width="474" alt="Screenshot 2024-03-14 at 00 34 16" src="https://github.com/pybricks/support/assets/63120157/f127bf1e-f15f-4b53-a3d9-1fc4e2d3fff4">

**Comment by johnmrichey** [comment]
I think the font is missing or being replaced by one that is not compatible.
![Uploading Screenshot 2024-03-14 at 00.34.16.png…]()

**Comment by laurensvalk** [comment]
Thanks for reporting. Apparently broken fonts is a common issue in MacOS Sonoma See e.g. https://www.reddit.com/r/MacOS/comments/16xmyfq/fonts_broken_in_safari_after_macos_sonoma_update/

I guess I'll have to find a way to reproduce it in order to fix it.

**Reply by laurensvalk** [reply]
@dlech, do you see broken class names as above, on your Mac? Thanks!

**Reply by dlech** [reply]
Chrome Version 122.0.6261.129 (Official Build) (arm64)

macOS Sonoma 14.4 (23E214)

v3.4.0 (Pybricks Beta v2.4.0-beta.7)

Looks normal to me.

**Reply by johnmrichey** [reply]
Chrome Version 122.0.6261.129 (Official Build) (arm64)
MacOS: 14.4 (23E214)

Pybricks Beta I don't know how to check that.
<img width="1710" alt="Screenshot 2024-03-14 at 22 37 51" src="https://github.com/pybricks/support/assets/63120157/8baccf6d-a095-4407-9265-35ab242d88f4">

**Reply by dlech** [reply]
> Pybricks Beta I don't know how to check that.

The version is in the about dialog.

**Comment by johnmrichey** [comment]
Any resolution to the font issue for Class naming please?
<img width="1624" alt="Screenshot 2024-03-22 at 14 07 55" src="https://github.com/pybricks/support/assets/63120157/1bc2a51d-76a8-44a3-bc2a-d4c0338bf32e">
<img width="1624" alt="Screenshot 2024-03-22 at 14 07 48" src="https://github.com/pybricks/support/assets/63120157/5956af0a-0ad0-4c35-b8f0-8a01f1e1d7b3">

**Comment by johnmrichey** [comment]
I found the fix from the link above and re-installed the four fonts and now it looks all good.

**Reply by laurensvalk** [reply]
For anyone running into this, can you link specifically to what solved it for you? Thanks!
